mbuild/formats/hoomd_snapshot.py

```diff
@@ -39,8 +51,12 @@ def to_hoomdsnapshot(structure,  ref_distance=1.0, ref_mass=1.0,
         largest mass value as ref_mass
         and largest epsilon value as ref_energy
     write_special_pairs : bool, optional, default=True
-        Writes out special pair information necessary to correctly use 
+        Writes out special pair information necessary to correctly use
         the OPLS fudged 1,4 interactions in HOOMD.
+    hoomd_snapshot : hoomd.data.SnapshotParticleData, optional, default=None
```

Comment:
Just to clarify, when this is provided a snapshot, the modified version of this snapshot is returned? With the structure object/etc added?.

Reply:
yes, if `hoomd_snapshot` is provided, then `make_snapshot` is not called and the objects from the parmed structure are added to the provided snapshot. The box from the parmed object will overwrite the box in the provided snapshot.

Reply:
awesome, thanks for the info!